### ISSUE TITLE
[FEATURE REQUEST] Allow options for which old school things are enabled?

### ISSUE THREAD

**Issue by TheTechRobo** [daattali/oldschool-github-extension#37]
For example, I've taken quite a liking to the rounded issue / PR labels, and the rounded avatars. So maybe have a settings menu that allows you to customize "what" features are modified?

**Comment by daattali** [comment]
Please explain the issue

**Comment by TheTechRobo** [comment]
I was editing my post while you typed that, accidentally pressed enter without explaining it lol

**Comment by TheTechRobo** [comment]
Edited post.

**Comment by daattali** [comment]
I was considering this initially (see here https://github.com/daattali/oldschool-github-extension/issues/16) but it looked like it wasn't going to be worth the time investment as most people didn't want that. I'd be happy for a PR if you'd like to contribute! But I don't think I'll personally find time to build that

**Comment by daattali** [comment]
I agree it can be nice, I just have over a dozen open source projects I actively maintain so I unfortunately have to be very selective in what features I dedicate time to

**Comment by TheTechRobo** [comment]
Ok! I'll check out the code and if I think I can do it, I'll try! Thanks for your answer.

Oh! Just saw your new comment 🙂.

> I just have over a dozen open source projects I actively maintain so I unfortunately have to be very selective in what features I dedicate time to

I understand. Thanks for your very quick answer, it shows a lot of dedication! :D

**Comment by daattali** [comment]
In #16 I linked to documentation for how to implement this in firefox and chrome if you want to give it a shot.
